arduino/include/arduino.h

```diff
@@ -0,0 +1,180 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+#include "serialib.h"
+#include <ios>
+#include <thread>
+
+class Arduino
+{
+public:
+  enum HEADERS
+  {
+    GET_FLOAT = 700,
+    GET_BOOLEAN = 701,
+    ASK_FOR_STATUS_PER_SENSOR = 30,
+    ASK_FOR_OVERALL_STATUS = 300,
+  };
+
+  enum ERROR_ARDUINO
+  {
+    SUCCSESS = 0,
+    ERROR_WITH_SERIALPORT,
+    ERROR_WITH_FILES,
+    THE_CHECKSUM_DID_NOT_WORK,
+    THE_ARDUINO_IS_NOT_RESPONDING,
+    BAD_STATUS,
+  };
+
+  /**
+   * @brief Construct a new Arduino::Arduino object
+   * 
+   */
+  Arduino(const std::string &portName, unsigned int buadRate, unsigned int timeout);
+
+  /*!
+  * @brief Destroy the Arduino::Arduino object 
+  */
+  ~Arduino();
+
+  /**
+   * @brief setter of the timeout
+   * 
+   * @param timeout int the time to abandon the serial communication
+   */
+  void setTimeout(int timeout);
+
+  /**
+   * @brief getter for the timeout of the serial port
+   * 
+   * @return int the time to abandon the serial communication
+   */
+  int getTimout();
+
+  /**
+   * @brief getter, will return an array of files that the class uses
+   * to output of the sensors
```

Comment:
```suggestion
   * @brief returns an array of files that the class uses
   * to output of the sensors
```
what are these files?

Reply:
fixed